src/idsk/components/intro-block/template.njk

```diff
@@ -0,0 +1,86 @@
+<div data-module="idsk-intro-block">
+    <div class="idsk-intro-block {{ params.classes if params.classes }}">
+        <div class=" {{ params.classContainer }}">
+            <div class="govuk-grid-column-full govuk-grid-column-two-thirds-from-desktop">
+                <h1 class="govuk-heading-xl">
+                    {{ params.headline if params.headline else "Ministerstvo investícií, regionálneho rozvoja a informatizácie Slovenskej republiky"}}
+                </h1>
+                <p class="idsk-intro-block-subtitle govuk-caption-l">Vitajte na stránke Ministerstva investícií, regionálneho rozvoja a informatizácie. Dozvedieť sa
```

Comment:
vsetky tie texty by mali byt jak parametre.. v pripade jazykovej mutaice to bude potrebne.. aj v pripade inych ministerstiev atd budu chiet mat vlastne texty... to plati napriec celemu bloku